### PR TITLE
fastRedstoneCrafting

### LINKED ISSUE #15: Fast redstone Crafting
Aggiungere crafting veloci agli elementi in redstone
image
image

### PR DESCRIPTION
**Pull request by DukeEdivad05**
Resolves #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,6 +300,14 @@ Ender Dragon always spawns a new dragon egg
 - Suggested options: `true`, `false`
 - Categories: `SURVIVAL`, `FEATURE`, `REDCRAFT`
 
+## fastRedstoneCrafting
+
+Aggiunge crafting alternativi di dispenser e repeater
```
Comment (BisUmTo) [inline README.md:305]: Questo è il file inglese...

Adds fast redstone crafting for Dispenser and Repeater.

**Review by BisUmTo — CHANGES_REQUESTED** [review]

**Comment by BisUmTo** [comment]
File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/advancements/fast_redstone_crafting/recipes/dispenser`

Line 55: 
```json
[
   "has_item_2",
   "has_the_recipe"
]
```

**Comment by BisUmTo** [comment]
File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/advancements/fast_redstone_crafting/fast_redstone_crafting.json`

Line 5:
```json
"text": "Some fast redstone crafting by BisUmTo, Gnottero & DukeEdivad05"
```

**Comment by BisUmTo** [comment]
File: carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/recipes/fast_redstone_crafting/dispenser/dispenser1.json

Line 21:
```json
   "item": "minecraft:dispenser"
}
```

Mal-formattazione del JSON, basta mettere a capo la parentesi graffa.

**Comment by BisUmTo** [comment]
File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/recipes/fast_redstone_crafting/dispenser/dispenser2.json`

Line 13:
```json
   "item": "minecraft:dispenser"
}
```

Mal-formattazione del JSON, basta mettere a capo la parentesi graffa.

**Comment by BisUmTo** [comment]
File: carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/functions/fast_redstone_crafting/uninstall.json

Line 10:
```json
datapack disable "file/RedPack - Fast Redstone Crafting"
```

Il datapack disabilitato deve coincidere con il nome del file

**Comment by DukeEdivad05** [comment]
> File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/advancements/fast_redstone_crafting/fast_redstone_crafting.json`
> 
> Line 5:
> ```json
> "text": "Some fast redstone crafting by BisUmTo, Gnottero & DukeEdivad05"
> ```
> 
> Mancano i tuoi crediti

Sei sicuro servano??

**Comment by DukeEdivad05** [comment]
> File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/advancements/fast_redstone_crafting/recipes/dispenser/dispenser1.json`
> 
> Line 55:
> 
> ```json
> [
>    "has_item_2",
>    "has_the_recipe"
> ]
> ```
> 
> Manca un criterio

Esiste un crafting delle stringhe??

**Comment by BisUmTo** [comment]
> > File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/advancements/fast_redstone_crafting/fast_redstone_crafting.json`
> > Line 5:
> > ```json
> > "text": "Some fast redstone crafting by BisUmTo, Gnottero & DukeEdivad05"
> > ```
> > 
> > 
> > Mancano i tuoi crediti
> 
> Sei sicuro servano??

Beh, sì...

**Comment by BisUmTo** [comment]
> > File: `carpet-redcraft-addons/src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Fast Redstone Crafting.zip/data/redpack/advancements/fast_redstone_crafting/recipes/dispenser/dispenser1.json`
> > Line 55:
> > ```json
> > [
> >    "has_item_2",
> >    "has_the_recipe"
> > ]
> > ```
> > 
> > 
> > Manca un criterio
> 
> Esiste un crafting delle stringhe??

`has_the_recipe` si riferisce al controllo sul dispenser stesso e non sull'ingrediente

**Review by BisUmTo — APPROVED** [review]

**Pull request merged**